providers/fireflies/handlers.go

```diff
@@ -262,3 +264,325 @@ func (c *Connector) parseReadResponse(
 		params.Fields,
 	)
 }
+
+// nolint:gocognit,cyclop,funlen
+func (c *Connector) buildWriteRequest(
+	ctx context.Context, params common.WriteParams,
+) (*http.Request, error) {
+	url, err := urlbuilder.New(c.ProviderInfo().BaseURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to build URL: %w", err)
+	}
+
+	recordData, err := common.RecordDataToMap(params.RecordData)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert record data to map: %w", err)
+	}
+
+	var mutation string
+
+	switch params.ObjectName {
+	case objectNameLiveMeeting:
+		if params.RecordId == "" {
+			meetingLink, ok := recordData["meeting_link"].(string)
+			if !ok {
+				return nil, ErrMeetingLinkRequired
+			}
+
+			mutation = fmt.Sprintf(`mutation {
```

Comment:
could you declare these as `graphql files` (maybe we can have conventions like `query_users.graphql` or `mutation_meeting.graphql`)